tests/test_user_config.py

```diff
@@ -24,8 +24,8 @@ def test_user_dir(tmpdir_factory, _cookiecutter_config_file):
 
 
         def test_valid_cookiecutter_config(_cookiecutter_config_file):
-            config_text = _cookiecutter_config_file.read()
-            config = poyo.parse_string(config_text)
+            with open(_cookiecutter_config_file) as f:
+                config = yaml.load(f, Loader=yaml.Loader)
```

Comment:
Shouldn't  `yaml.safe_load` nbe used instead of `yaml.load`?

Reply:
Would be `yaml.SafeLoader`, but I'm not sure it matters since the config file fixture is created by the test: it's not from a third-party.